Description: Do not write timestamps in documentation generated by Doxygen
 In order to make the build reproducible, we configure Doxygen to skip
 writing timestamps in the HTML documentation it generates.
Author: Jérémy Bobbio <lunar@debian.org>
Last-Update: 2014-09-24

--- cwidget-0.5.17.orig/Doxyfile.in
+++ cwidget-0.5.17/Doxyfile.in
@@ -699,6 +699,15 @@ HTML_HEADER            =
 
 HTML_FOOTER            = 
 
+# If the HTML_TIMESTAMP tag is set to YES then the footer of each generated HTML
+# page will contain the date and time when the page was generated. Setting this
+# to NO can help when comparing the output of multiple runs.
+# The default value is: YES.
+# This tag requires that the tag GENERATE_HTML is set to YES.
+
+HTML_TIMESTAMP         = NO
+
+
 # The HTML_STYLESHEET tag can be used to specify a user-defined cascading 
 # style sheet that is used by each HTML page. It can be used to 
 # fine-tune the look of the HTML output. If the tag is left blank doxygen 
